plugins/cost-insights/api-report.md

```diff
@@ -370,10 +370,13 @@ export const costInsightsApiRef: ApiRef<CostInsightsApi>;
 // @public (undocumented)
 export const CostInsightsLabelDataflowInstructionsPage: () => JSX.Element;
 
+// Warning: (ae-forgotten-export) The symbol "CostInsightsPageProps" needs to be exported by the entry point index.d.ts
```

Comment:
Probably makes sense to export `CostInsightsPageProps` so that anyone constructing it to pass in can strongly type it.